src/screens/MovieDetailScreen.tsx

```diff
@@ -1,16 +1,372 @@
-import * as React from 'react';
-import {Text, View, StyleSheet} from 'react-native';
+import {useState, useEffect} from 'react';
+import _ from 'lodash';
+import {
+  Text,
+  View,
+  StyleSheet,
+  StatusBar,
+  ScrollView,
+  ActivityIndicator,
+  ImageBackground,
+  Image,
+  FlatList,
+  TouchableOpacity,
+} from 'react-native';
+import axios from 'axios';
+import LinearGradient from 'react-native-linear-gradient';
 
-interface Props {}
+import {
+  movie_details,
+  movie_cast_details,
+  base_image_path,
+} from '../api/apicalls';
+import {
+  COLORS,
+  FONTFAMILY,
+  FONTSIZE,
+  SPACING,
+  BORDERRADIUS,
+} from '../theme/theme';
+import Header from '../components/Header';
+import CustomIcon from '../components/CustomIcon';
+import CastCard from '../components/CastCard';
 
 const styles = StyleSheet.create({
```

Comment:
The file have exceeded `350 lines of code` and expected to get more messy with time. 
Transfer Internal stylesheet to external stylesheet.